templates/helm/values.schema.json

```diff
@@ -0,0 +1,213 @@
+        "containerPort": {
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 65535
+        },
+        "nodeSelector": {
+          "type": "object"
+        },
+        "tolerations": {
+          "type": "object"
+        },
+        "affinity": {
+          "type": "object"
+        },
+        "priorityClassName": {
+          "type": "string"
+        }
+      },
+      "required": [
+          "containerPort",
+          "nodeSelector"
+      ],
+      "type": "object"
+    },
+    "metrics": {
+      "description": "Metrics settings",
+      "properties": {
+        "service": {
+          "description": "Kubernetes service settings",
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "type": {
+              "type": "string",
+              "enum": ["ClusterIP", "NodePort", "LoadBalancer", "ExternalName"]
+            }
+          },
+          "required": [
+              "create",
+              "type"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+          "service"
+      ],
+      "type": "object"
+    },
+    "resources": {
+      "description": "Kubernetes resources settings",
+      "properties": {
+        "requests": {
+          "description": "Kubernetes resource requests",
+          "properties": {
+            "memory": {
+              "oneOf": [
+                { "type": "number" },
+                { "type": "string" }
+              ]
+            },
+            "cpu": {
+              "oneOf": [
+                { "type": "number" },
+                { "type": "string" }
+              ]
+            }
+          },
+          "required": [
+              "memory",
+              "cpu"
+          ],
+          "type": "object"
+        },
+        "limits": {
+          "description": "Kubernetes resource limits",
+          "properties": {
+            "memory": {
+              "oneOf": [
+                { "type": "number" },
+                { "type": "string" }
+              ]
+            },
+            "cpu": {
+              "oneOf": [
+                { "type": "number" },
+                { "type": "string" }
+              ]
+            }
+          },
+          "required": [
+              "memory",
+              "cpu"
+          ],
+          "type": "object"
+        }
+      },
+      "required": [
+          "requests",
+          "limits"
+      ],
+      "type": "object"
+    },
+    "aws": {
+      "description": "AWS API settings",
+      "properties": {
+        "region": {
+          "type": "string"
+        },
+        "endpoint": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "log": {
+      "description": "Logging settings",
+      "properties": {
+        "enable_development_logging": {
+          "type": "boolean"
+        },
+        "level": {
+          "type": "string",
+          "enum": ["debug", "info"]
```

Comment:
There may be more levels. Perhaps `error` and `fatal`? For now I might just recommend leaving it as an open-ended string.